docs/platforms/platforms_index.rst

```diff
@@ -134,6 +134,36 @@ available nodes where possible. ``jsrun`` can also queue runs. However, on
 other cluster and multi-node systems, if auto-resources is disabled, then runs without
 a hostlist or machinefile supplied may be undesirably scheduled to the same nodes.
 
+Zero-resource workers
+~~~~~~~~~~~~~~~~~~~~~
+
+Users with persistent ``gen_f`` functions may notice with certain
+configurations that associated persistent workers are still automatically assigned
```

Comment:
Wondering if can simplify the sentence. I'm wondering are there configurations where this is not the case? Could it be just: 

Users with persistent ``gen_f`` functions may notice that the persistent workers are still automatically assigned....